### PR TITLE
feat(css): should remove `at-import` statements in output

### PR DESCRIPTION
**Pull request by hyf0**

### CODE REVIEW

**Comment by hyf0** [comment]
* **#2424** <a href="https://app.graphite.dev/github/pr/rolldown/rolldown/2424?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2423** <a href="https://app.graphite.dev/github/pr/rolldown/rolldown/2423?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#2422** <a href="https://app.graphite.dev/github/pr/rolldown/rolldown/2422?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2420** <a href="https://app.graphite.dev/github/pr/rolldown/rolldown/2420?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2419** <a href="https://app.graphite.dev/github/pr/rolldown/rolldown/2419?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @hyf0 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: crates/rolldown/src/css/mod.rs [hunk 1]
```diff
@@ -28,10 +30,25 @@ pub fn create_css_view(
           SymbolRef::from((ModuleIdx::from_raw(0), SymbolId::from_usize(0))),
           range.start,
         ));
+        let mut range_end = range.end as usize;
+        if source.is_char_boundary(range_end) {
+          let next_char = source[range_end..].chars().next();
```
Comment (IWANABETHATGUY) [inline crates/rolldown/src/css/mod.rs]: `starts_with` should be better, since `\n` is just a ascii char

File: crates/rolldown/src/css/mod.rs [hunk 2]
```diff
@@ -28,10 +30,25 @@ pub fn create_css_view(
...
+          if next_char == Some('\n') {
```
Comment (IWANABETHATGUY) [inline crates/rolldown/src/css/mod.rs]: what about `\r\n`?

**Comment by IWANABETHATGUY** [comment]
sorry,  I accidentally merge a middle pull request in stacks.

File: crates/rolldown/src/css/mod.rs (hunk 1 above)
Comment (hyf0) [reply crates/rolldown/src/css/mod.rs]: Done

File: crates/rolldown/src/css/mod.rs (hunk 2 above)
Comment (hyf0) [reply crates/rolldown/src/css/mod.rs]: Added.

**Review by IWANABETHATGUY — APPROVED** [review]

**Pull request merged**